src/tools/info/Makefile.am

```diff
@@ -8,6 +9,10 @@ bin_PROGRAMS          = ucx_info
 AM_CPPFLAGS           = \
     -I$(abs_top_srcdir)/src \
     -I$(abs_top_builddir)/src
+if HAVE_IB
+AM_CPPFLAGS += $(IBVERBS_CPPFLAGS)
```

Comment:
I did for the LDFLAGS, they get inherited from libtool. Now I think there is no other choices for CPPFLAGS.